TODO/core-plot-tutorial-getting-started.md

```diff
@@ -273,13 +273,13 @@ First, replace current `numberOfRecordsForPlot(_:)` with the following:
 
 
 
-This method determines the number of slices to show on the graph; it will display one pie slice for each symbol.
+这个方法决定了图表的分片数量，它将为每个标记展示一个饼图分片。
```

Comment:
> This method determines the number of slices to show on the graph; it will display one pie slice for each symbol.
> 这个方法决定了图表的分片数量，它将为每个标记展示一个饼图分片。

这个方法决定了有多少块显示在饼状图上， 它将为每一个标记显示一块。 

----------
大饼应该是一块一块的吧？ 哈哈 。